atomicapp/utils.py

```diff
@@ -267,22 +267,23 @@ def run_cmd(cmd, checkexitcode=True, stdin=None):
         return ec, stdout, stderr
 
     @staticmethod
-    def askFor(what, info):
+    def askFor(what, info, app_name):
         repeat = True
         desc = info["description"]
+        logger.debug(info)
         constraints = None
         if "constraints" in info:
             constraints = info["constraints"]
         while repeat:
             repeat = False
             if "default" in info:
                 value = raw_input(
-                    "ANSWER >> %s (%s, default: %s): " % (what, desc, info["default"]))
+                    "ANSWER => %s | %s (%s, default: %s): " % (app_name, what, desc, info["default"]))
                 if len(value) == 0:
                     value = info["default"]
             else:
                 try:
-                    value = raw_input("ANSWER >> %s (%s): " % (what, desc))
```

Comment:
wonder why this `raw_input()` call needs a try/except and the other one doesn't?